### PR TITLE
Update dashboards from spreadsheet

### PR DESCRIPTION
**Pull request by mattrco**
Allows dashboards to be updated from the spreadsheet.

Improves tests and handles various error cases better.

Fixes an attribute on modules to ensure dataset is filtered.

https://www.agileplannerapp.com/boards/15088/cards/9457
https://www.agileplannerapp.com/boards/15088/cards/9454

### CODE REVIEW

**Comment by mattrco** [comment]
test

File: stagecraft/tools/fixtures/names.json [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-    "BIS",
-    "Advisory, Conciliation and Arbitration Service",
-    "Acas elearning registrations",
-    "",
-    "Training and resources on workplace relations",
-    "/training-resources-on-workplace-relations",
-    "Registrations to use online training and resources on workplace relations",
-    "/registrations",
-    "",
-    "Data for registrations to use Acas's elearning modules on employment relations.",
```
Comment (mattrco) [inline stagecraft/tools/fixtures/names.json]: test

File: stagecraft/tools/txex-migration.py [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-
-import os
-import sys
-import redirects
-
-try:
-    username = os.environ['GOOGLE_USERNAME']
-    password = os.environ['GOOGLE_PASSWORD']
-except KeyError:
-    print("Please supply username (GOOGLE_USERNAME)"
```
Comment (jcbashdown) [inline stagecraft/tools/txex-migration.py]: test

File: stagecraft/tools/test_import_dashboards.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
+import json
+
+from mock import patch, Mock
+from nose.tools import nottest
```
Comment (jcbashdown) [inline stagecraft/tools/test_import_dashboards.py]: Still needed?

**Pull request merged**